sample-xml/app/src/main/java/co/nimblehq/sample/xml/ui/screens/second/SecondViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.sample.xml.ui.screens.second
+
+import co.nimblehq.sample.xml.ui.base.BaseViewModel
+import co.nimblehq.sample.xml.util.DispatchersProvider
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class SecondViewModel @Inject constructor(
```

Comment:
Should we leave out `SecondViewModel.kt` & `SecondFragment.kt` from part 1? (So add it to part 2)

Reply:
@Tuubz Why we should not add it here? Just curious?

Reply:
Oh, if I'm not mistaken it's not being used at all at the moment. That's why 🙏

Reply:
@Tuubz And I also added the `TODO` comment to make the code reviewer know what the next plan is. So I don't think I need to remove it. 🙏🏻